package.json

```diff
@@ -33,79 +33,75 @@
-		"@faker-js/faker": "^7.3.0",
-		"@testing-library/jest-dom": "^5.16.4",
+		"@babel/core": "^7.20.12",
+		"@babel/plugin-proposal-class-properties": "^7.18.6",
+		"@babel/plugin-transform-runtime": "^7.19.6",
+		"@babel/preset-env": "^7.20.2",
+		"@babel/preset-react": "^7.18.6",
+		"@babel/preset-typescript": "^7.18.6",
+		"@commitlint/cli": "^17.4.3",
+		"@commitlint/config-conventional": "^17.4.3",
+		"@faker-js/faker": "^7.6.0",
+		"@testing-library/jest-dom": "^5.16.5",
 		"@testing-library/react": "^12.1.5",
 		"@testing-library/react-hooks": "^8.0.1",
-		"@testing-library/user-event": "^14.3.0",
-		"@types/history": "4.7.9",
+		"@testing-library/user-event": "^14.4.3",
 		"@types/jest": "^28.1.6",
-		"@types/lodash": "4.14.177",
-		"@types/react": "17.0.36",
-		"@types/react-dom": "17.0.11",
-		"@types/react-redux": "^7.1.20",
-		"@types/react-router-dom": "5.3.2",
-		"@types/styled-components": "^5.1.15",
+		"@types/lodash": "^4.14.191",
+		"@types/react": "^17.0.53",
+		"@types/react-redux": "^7.1.25",
+		"@types/react-router-dom": "^5.3.3",
+		"@types/styled-components": "^5.1.26",
 		"@zextras/carbonio-ui-configs": "^0.1.11",
-		"@zextras/carbonio-ui-sdk": "1.5.0",
-		"babel-jest": "26.6.3",
-		"babel-plugin-i18next-extract": "0.8.3",
-		"babel-plugin-styled-components": "2.0.1",
-		"eslint-plugin-jest-dom": "^4.0.2",
+		"@zextras/carbonio-ui-sdk": "^1.5.0",
+		"babel-plugin-i18next-extract": "^0.9.0",
+		"babel-plugin-styled-components": "^2.0.7",
+		"eslint-plugin-jest-dom": "^4.0.3",
 		"eslint-plugin-notice": "^0.9.10",
-		"eslint-plugin-testing-library": "^5.5.1",
+		"eslint-plugin-testing-library": "^5.10.1",
 		"eslint-plugin-unused-imports": "^2.0.0",
-		"husky": "^5.2.0",
-		"is-ci": "3.0.1",
-		"jest": "^28.1.3",
-		"jest-environment-jsdom": "^28.1.3",
-		"jest-fail-on-console": "^2.4.2",
+		"husky": "^8.0.3",
+		"is-ci": "^3.0.1",
+		"jest": "^29.4.2",
+		"jest-environment-jsdom": "^29.4.2",
+		"jest-fail-on-console": "^3.0.2",
 		"jest-fetch-mock": "^3.0.3",
-		"jest-junit": "^14.0.0",
-		"jest-styled-components": "^7.0.6",
-		"pinst": "2.1.1",
+		"jest-junit": "^15.0.0",
+		"jest-styled-components": "^7.1.1",
+		"msw": "^1.0.1",
+		"pinst": "^3.0.0",
 		"ts-node": "^10.9.1"
 	},
 	"dependencies": {
 		"@emotion/react": "^11.10.4",
 		"@emotion/styled": "^11.10.4",
 		"@mui/icons-material": "^5.10.6",
 		"@mui/material": "^5.10.6",
-		"@reduxjs/toolkit": "^1.6.2",
+		"@reduxjs/toolkit": "~1.7.2",
 		"@testing-library/react-hooks": "^8.0.1",
-		"@zextras/carbonio-design-system": "1.1.0",
-		"@zextras/carbonio-shell-ui": "1.1.0",
-		"@zextras/carbonio-ui-preview": "1.0.0",
-		"core-js": "3.19.1",
+		"@zextras/carbonio-design-system": "^1.1.0",
+		"@zextras/carbonio-shell-ui": "^1.1.1",
+		"@zextras/carbonio-ui-preview": "^1.0.0",
+		"core-js": "^3.27.2",
 		"date-arithmetic": "^4.1.0",
-		"formik": "2.1.4",
-		"i18next": "^21.5.4",
-		"immer": "7.0.14",
+		"i18next": "^21.10.0",
```

Comment:
i18next latest is 22.4.9

Reply:
it is shimmed by the shell, version ^21.10.0 is correct